cmd/erasure-server-pool-decom.go

```diff
@@ -778,6 +778,11 @@ func (z *erasureServerPools) decommissionPool(ctx context.Context, idx int, pool
 					continue
 				}
 
+				if bi.Name == minioMetaBucket && strings.Contains(version.Name, dataUsageCacheName) {
```

Comment:
Not sure I understand why ? the code is not ready to decomissions data usage cache data ?

Reply:
yes, why would we need to do it? @vadmeste This file is autogenerated, it is used as a temporary value until we write usage.json. There is no reason to move it.